README.md

```diff
@@ -1536,7 +1536,7 @@ You can refer to the example in this
 [guide](https://cloud.google.com/identity-platform/docs/multi-tenancy-ui).
 
 There is also a
-[quickstart](https://github.com/firebase/quickstart-js/blob/master/auth/multi-tenant-ui.html)
+[quickstart](https://cloud.google.com/identity-platform/docs/multi-tenancy-quickstart)
```

Comment:
Why did you change this? Can you revert it? This is the GitHub demo app that the explanation refers to.